zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Root.java

```diff
@@ -126,15 +85,61 @@ public static void init() {
             Collections.CollectionList collections = zebedee.getCollections().list();
             alertOnInProgressCollections(collections, notifier);
             loadExistingCollectionsIntoScheduler(collections);
-        } catch (IOException e) {
-            final String message = "failed to load collections list on startup";
-            error().logException(e, message);
-            throw new RuntimeException(message, e);
+        } catch (IOException ex) {
+            throw new RuntimeException("failed to load collections list on startup", ex);
         }
 
         info().data(ZEBEDEE_ROOT, rootDir).log("zebedee cmd initialization completed successfully");
     }
 
+    /**
+     * Validate the zebedee root path meets the requirements - not null/empty, the file exists and is a directory.
+     * Throws {@link RuntimeException} if any of the checks fail.
+     *
+     * @param rootDir the path to validate.
+     * @return the root dir value as a {@link Path} object.
+     */
+    private static Path validateZebedeeRootConfig(String rootDir) {
+        if (StringUtils.isEmpty(rootDir)) {
+            throw new RuntimeException("zebedee_root env var required but none provided");
+        }
+
+        Path p = Paths.get(rootDir);
+        if (Files.notExists(p)) {
+            throw new RuntimeException(
+                    format("invalid zebedee_root config: {0} directory does not exist", rootDir));
+        }
+
+        if (!Files.isDirectory(p)) {
+            throw new RuntimeException(
+                    format("invalid zebedee_root config:{0} the specified path is not a directory", rootDir));
+        }
+
+        info().data(ZEBEDEE_ROOT, rootDir)
+                .log("successfully validated zebedee_root config proceeding beginning CMS initialization");
+        return p;
+    }
```

Comment:
just a thought, I was wondering if you check for one or more of the known directories here as Zebedee will start up if the env variable to set to an empty directory ?